states/scene.js

```diff
@@ -27,21 +27,12 @@ module.exports = {
     this.dialogueGroup = this.add.group();
   },
 
-  setupAudio() {
-    this.music = this.add.audio('scene-soundtrack');
-  },
-
   startScene() {
-    this.music.play();
-
     this.enableInput();
-
     this.playDialogue(0);
   },
 
   endScene() {
-    this.sound.destroy();
```

Comment:
Make sure this does get called at the end of the scene, so the assets are cleaned up, to avoid memory leaks. Sounds are the only thing (I think) not cleaned up by a state change.